refill-style.yaml

```diff
@@ -937,6 +807,9 @@ styles:
                                     color.rgb,
                                     step(.5,dot(vec3(0., 0., 1.), worldNormal())));
 
+    no_waves_playa:
```

Comment:
We should eliminate this style since it doesn't do anything (besides just render regular polygons).

Reply:
If I don't use this, playa drawes the waves shader, I need to turn off the waves shader here

Reply:
Is there somewhere else the `no_waves_playa` style is further defined? Otherwise you should be able to just set `style: polygons` in places where you would use it instead.

Reply:
ok, fixed it
noting here: will have to make these changes to  no-texture.yaml